cmd/ping.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/heartbeatctl/pkg/client"
+	"github.com/giantswarm/heartbeatctl/pkg/cmdutil"
+	"github.com/giantswarm/heartbeatctl/pkg/ctl"
+)
+
+// pingCmdOptions holds values for options accepted by the ping command
+type pingCmdOptions struct {
+	selectorOptions *cmdutil.SelectorOptions
+}
+
+var (
+	pingDocLong = heredoc.Doc(`
+		Ping specified heartbeats to remove their expiration status.
```

Comment:
We are not removing anything here, this feels wrong. I would prefer something like this :

```suggestion
		Issue a ping request to specified heartbeats.
```

Reply:
Fixed in https://github.com/giantswarm/heartbeatctl/pull/15